frontend/src/components/Thread.vue

```diff
@@ -53,16 +69,34 @@ export default {
     },
   },
   methods: {
+    // TODO: this is currently not used, because I can't figure out a good place to put this
     resolve(thread) {
       this.$store.commit('resolveThread', thread);
     },
     reply(thread) {
-      console.log(thread);
-      this.$store.commit('addMessageToThread', [this.newMessages[thread.id], thread]);
-      this.newMessages[thread.id] = "";
+      if (this.newMessages[thread.id]) {
+        console.log(thread);
+        this.$store.commit('addMessageToThread', [this.newMessages[thread.id], thread]);
+        this.newMessages[thread.id] = "";
+      } else {
+        console.log("Reply is empty!");
+        // TODO: give "text is empty" error
```

Comment:
do we still need this todo? what are we doing if the reply is empty? just nothing right? i think that's fine.

Reply:
Right now it's just nothing, but I would prefer alerts like #87